lib/dom-parser.js

```diff
@@ -242,10 +247,20 @@ function appendElement (hander,node) {
     }
 }//appendChild and setAttributeNS are preformance key
 
-//if(typeof require == 'function'){
-var htmlEntity = require('./entities');
-var XMLReader = require('./sax').XMLReader;
-var DOMImplementation = exports.DOMImplementation = require('./dom').DOMImplementation;
-exports.XMLSerializer = require('./dom').XMLSerializer ;
+// Primary Web API interfaces
 exports.DOMParser = DOMParser;
-//}
+exports.XMLSerializer = DOM.XMLSerializer;
+// Level 2 Core DOM interfaces:
```

Comment:
[Cleanup] I would add a space here, for the sake of easier readability

```suggestion
exports.XMLSerializer = DOM.XMLSerializer;

// Level 2 Core DOM interfaces:
```

Reply:
Do you want me to clean up the formatting? The existing code base had mixed tabs and spaces, and I initially cleaned all that up, but was concerned it would make the PR harder to review.